src/Couchbase.Lite.Tests.Shared/CSharpTest.cs

```diff
@@ -670,7 +671,21 @@ public async Task TestMainThreadScheduler()
 
         #endif
 
-        
+        [Fact]
+        public void TestCBDebugItemsMustNotBeNull()
+        {
+            List<object> list = new List<object>();
+            list.Add("couchbase");
+            list.Add(null);
+            list.Add("debug");
+            Action badAction = (() =>
+            CBDebug.ItemsMustNotBeNull(
+                WriteLog.To.Query, 
+                nameof(CSharpTest), 
+                nameof(TestCBDebugItemsMustNotBeNull), 
+                list));
+            badAction.ShouldThrow<ArgumentNullException>("because the item in enumeration cannot be null.");
```

Comment:
Add a list without `null` to make sure that no exception is thrown (no asserts needed)